File: 2C++/Product.cpp
class Product{
    //atribut
    private:
        int price;
        string IdProduct;

     //konstruktor 
    public: 
        Product(){
            this->price = 0;
            this->IdProduct = "";
        }

    
    //setter & getter price
    void setprice(int price){
        this->price = price;
    }
    int getprice(){
        return this->price;
    }

    //setter & getter IdProduct  
    void setIdProduct(string IdProduct){
        this->IdProduct = IdProduct;
    }
    string getIdProduct(){
        return this->IdProduct;
    }

    ~Product(){
    }
};

File: 1Java/Product.java
public class Product{
    //atribut
    private int price;
    private String IdProduct;
     
    //konstruktor 
    public Product(){
    }

    //setter & getter price
    public void setprice(int price){
        this.price = price;
    }
    public int getprice(){
        return this.price;
    }

    //setter & getter IdProduct
    public void setIdProduct(String IdProduct){
        this.IdProduct = IdProduct;
    }
    public String getIdProduct(){
        return this.IdProduct;
    }
}
